docs/react/introduce.en-US.md

```diff
@@ -0,0 +1,95 @@
+---
+order: 0
+title: Ant Design of React
+---
+
+antd is a set of React components which follow Ant Design specification. It is designed to develop RIA sucn as dashboard to serve enterprise.
+
+<div class="pic-plus">
+  <img width="150" src="https://t.alipayobjects.com/images/rmsweb/T11aVgXc4eXXXXXXXX.svg">
+  <span>+</span>
+  <img width="160" src="https://t.alipayobjects.com/images/rmsweb/T16xRhXkxbXXXXXXXX.svg">
+</div>
+
+<style>
+.pic-plus > * {
+  display: inline-block!important;
+  vertical-align: middle;
+}
+.pic-plus span {
+  font-size: 30px;
+  color: #aaa;
+  margin: 0 20px;
+}
+</style>
+
+---
+
+## Features
+
+- Designed as Ant Design, a design language to create user friendly and beautiful websites.
```

Comment:
- Using Ant Design, a design language for creating user friendly and beautiful websites.